--- libtheora-1.0alpha2/configure.in.orig	2003-06-09 23:23:50.000000000 +0200
+++ libtheora-1.0alpha2/configure.in	2003-08-16 02:11:46.770106256 +0200
@@ -79,7 +79,7 @@
 
 AM_PATH_SDL(,[
 	HAVE_SDL=yes
-	SDL_LIBS=`$SDL_CONFIG --static-libs`
+	SDL_LIBS=`$SDL_CONFIG --libs`
 ],AC_MSG_WARN([*** Unable to find SDL -- Not compiling player_example ***]))
 AM_CONDITIONAL(HAVE_SDL, test "$HAVE_SDL" = "yes")
 
--- libtheora-1.0alpha2/examples/Makefile.am.orig	2003-06-04 00:14:56.000000000 +0200
+++ libtheora-1.0alpha2/examples/Makefile.am	2003-08-16 02:15:48.466362840 +0200
@@ -12,7 +12,6 @@
 
 noinst_PROGRAMS = encoder_example dump_video $(player_example)
 
-LDFLAGS = -all-static 
 LDADD = ../lib/libtheora.la -logg -lm
 
 dump_video_SOURCES = dump_video.c
